docs/self-hosting.md

```diff
@@ -56,6 +59,11 @@ Before you begin, make sure you have the following installed on your system:
 
 - [Docker](https://docs.docker.com/engine/install/)
 
+### Limitations
+
+- The Docker image is not suitable for production use due to limitations in the configuration.
+- The Docker image only supports amd64 architecture.
+
```

Comment:
This can be inside a [container](https://vitepress.dev/guide/markdown#custom-containers) like `warning`